src/lashup_kv.erl

```diff
@@ -43,7 +44,8 @@
 -define(REJECT_OBJECT_SIZE_KB, 100).
 
 -record(state, {
-  mc_ref = erlang:error() :: reference()
+  mc_ref = erlang:error() :: reference(),
+  nclock = [] :: riak_dt_vclock:vclock()
```

Comment:
why use a vclock here, when you're not doing any vclock ops? Maybe something like a map, or ordsets?